### PR TITLE
move to eslint

### PR DESCRIPTION
**Pull request by jdx**
one day when we have a plugin generator I plan to export a configuration like this

### CODE REVIEW

File: src/namespaces.js [hunk 1]
```diff
@@ -45,28 +47,28 @@ export default class {
...
 
-  static _namespacePlugin (namespace: ?string, plugin: Object) : Object {
+  static _namespacePlugin (namespace: ?string, plugin: Object): Object {
     if (!namespace) return plugin
-    let nplugin = {namespace}
+    let nplugin: Object = {namespace}
     nplugin.commands = plugin.commands.map(cmd => {
       return {
-        topic: `${namespace}:${cmd.topic}`,
+        topic: `${namespace || ''}:${cmd.topic}`,
```
Comment (RasPhilCo) [inline src/namespaces.js]: flow shouldn't require this since there is a guard up top, but alas...

Comment (RasPhilCo) [reply src/namespaces.js]: p.s. this all goes away 😉

Comment (jdx) [reply src/namespaces.js]: ah I can fix that

**Review by RasPhilCo — APPROVED** [review]
🏄

**Pull request merged**